editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditFeaturesWithFeatureLinkedAnnotationSample.java

```diff
@@ -0,0 +1,288 @@
+            geodatabase.getGeodatabaseAnnotationTable("Loudoun_Address_PointsAnno_1"));
+          var parcelLinesAnnotationLayer = new AnnotationLayer(
+            geodatabase.getGeodatabaseAnnotationTable("ParcelLinesAnno_1"));
+
+          // add the feature layers and annotation layers to the map
+          map.getOperationalLayers().addAll(Arrays.asList(
+            addressPointFeatureLayer, parcelLinesFeatureLayer, addressPointsAnnotationLayer, parcelLinesAnnotationLayer));
+        } else {
+          // show alert if geodatabase fails to load
+          new Alert(Alert.AlertType.ERROR, "Error loading Geodatabase.").show();
+        }
+      });
+
+      // select the nearest feature from where the user clicked, or move the selected feature to the given screen point
+      mapView.setOnMouseClicked(event -> {
+        if (event.isStillSincePress() && event.getButton() == MouseButton.PRIMARY) {
+          // create a point where the user clicked
+          Point2D screenPoint = new Point2D(event.getX(), event.getY());
+          // if a feature hasn't been selected, select the feature
+          if (selectedFeature == null) {
+            identifyFeature(screenPoint);
+          } else {
+            // convert the screen point to a map point
+            Point mapPoint = mapView.screenToLocation(screenPoint);
+
+            // if the feature is a polyline, move the polyline
+            if (selectedFeature.getGeometry().getGeometryType() == GeometryType.POLYLINE) {
+              // get the selected feature's geometry as a polyline
+              Polyline polyline = (Polyline) selectedFeature.getGeometry();
+
+              // create a polyline builder to add and remove parts from the polyline
+              PolylineBuilder polylineBuilder = new PolylineBuilder(polyline);
+
+              // get the nearest vertex to the map point on the polyline
+              ProximityResult nearestVertex =
+                GeometryEngine.nearestVertex(polyline, (Point) GeometryEngine.project(mapPoint, polyline.getSpatialReference()));
+
+              // get the part of the polyline nearest to the map point
+              Part part = polylineBuilder.getParts().get(Math.toIntExact(nearestVertex.getPartIndex()));
+
+              // remove the nearest point to the map point from the part
+              part.removePoint((int) nearestVertex.getPointIndex());
+
+              // add the map point as the new point on the part
+              part.addPoint((Point) GeometryEngine.project(mapPoint, polyline.getSpatialReference()));
+
+              // set the selected feature's geometry to the new polyline
+              selectedFeature.setGeometry(polylineBuilder.toGeometry());
+
+              // if the feature is a point, move the point
+            } else if (selectedFeature.getGeometry().getGeometryType() == GeometryType.POINT) {
+              // set the selected features' geometry to a new map point
```

Comment:
correcting apostrophe
```suggestion
              // set the selected feature's geometry to a new map point
```